clients/ts/signalr/src/HttpClient.ts

```diff
@@ -84,15 +84,24 @@ export class DefaultHttpClient extends HttpClient {
                 xhr.timeout = request.timeout;
             }
 
-            xhr.onload = () => {
-                if (request.abortSignal) {
-                    request.abortSignal.onabort = null;
-                }
-
-                if (xhr.status >= 200 && xhr.status < 300) {
-                    resolve(new HttpResponse(xhr.status, xhr.statusText, xhr.response || xhr.responseText));
-                } else {
-                    reject(new HttpError(xhr.statusText, xhr.status));
+            xhr.onreadystatechange = () => {
+                this.logger.log(LogLevel.Trace, `onreadystatechange(${xhr.readyState}:${xhr.status}:${xhr.statusText})`);
```

Comment:
I'd like the logging to be a bit friendlier, e.g. map ready state numbers to text - https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/readyState

Something like "HTTP request state changed. New state '${readStateDescription}'. HTTP Status: ${status}: ${statusText}".

Reply:
I don't want to map it to text, because we don't compare it to text. It needs to be the raw data. The only reason I left this one in here was for our own debugging. I'd be ok removing it.